### ISSUE TITLE
davx⁵ Address Book local storage error - couldn't apply batch operation

### ISSUE THREAD

**Issue by edddiiie** [bitfireAT/davx5-manual#6]
Before posting I spent the entire afternoon looking for a solution. I did pay a contribution by purchasing from Google Play Store so I was hoping it will be easy to setup. I am running CPanel and finally found how to configure the App in the Docs. The Calendar work but getting this error for Contacts. There is a similar issue but for Tasks but I can't figure it out, other that possibly field mismatch. I am using the App on Oppo Android 12.

Not sure if it is related but I had the App setup initially with the same account user on CPanel then had to Clear Data, Clear Cache and start over again when I found the documentation on how to setup with CPanel. Could I have left some orphaned Objects that are conflicting? How should I proceed, a full uninstall and re-install or is there another issue that you can see?

**Comment by edddiiie** [comment]
I want to add an observation. On CPanel, if I share the Address Book of the master account with say user@mydomain.com and on Android App I disable the Address Book for user@mydomain.com and only leave the master address book it work OK. On Android App I am logged in the service on CPanle as user@mydomain.com. Does that give a clue?

**Comment by edddiiie** [comment]
Last post for today, It's been hours since I started with this and I am at a point where I can leave it aside. What I did was to delete the [user@mydomain.com] address book that was not syncing and created a brand new address book for this user, this resolved the error message. Looks like it was some server setting or corrupt address book. I will continue with it tomorrow now that I know it is working without a single error.
